core/src/main/java/nl/nn/adapterframework/jdbc/migration/BytesResourceAccessor.java

```diff
@@ -17,42 +17,62 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
 import java.util.SortedSet;
+import java.util.TreeSet;
 
 import liquibase.resource.InputStreamList;
 import liquibase.resource.ResourceAccessor;
+import nl.nn.adapterframework.core.Resource;
+import nl.nn.adapterframework.util.LogUtil;
 
 /**
  * @author alisihab
  *
  */
-public class StreamResourceAccessor implements ResourceAccessor {
+public class BytesResourceAccessor implements ResourceAccessor {
```

Comment:
Deze class heet nu BytesResourceAccessor, maar is eigenlijk ResourceResourceAccessor.